sdk/maps/azure-maps-search/samples/common/common.py

```diff
@@ -0,0 +1,27 @@
+from azure.core.credentials import AzureKeyCredential
+from azure.core.pipeline._base import SansIOHTTPPolicy
+from azure.core.pipeline.transport._base import HttpRequest
+from azure.core.pipeline import PipelineRequest
+
+class AzureKeyInQueryCredentialPolicy(SansIOHTTPPolicy):
```

Comment:
This should not be in a sample.
Customers should never need to define their own credential policies, and neither our SDKs nor our customers should ever import from private/internal modules in azure.core.

Reply:
Removed unused and inappropriate methods.